app/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.7'
 services:
   search-api:
     build:
-      context: .
+      context: SearchApi
```

Comment:
If possible, I'd suggest moving the docker-compose file to the parent directory. Reason being is that we may introduce other components of the fams3 stack that won't fit in the `app` folder. Plus, I think its a common practice to put it at the top level directory.

Reply:
agree